src/main/java/uk/gov/register/resources/ItemResource.java

```diff
@@ -25,14 +27,14 @@ public ItemResource(RegisterReadOnly register, ViewFactory viewFactory) {
     @Path("/{item-hash}")
     @Produces({ExtraMediaType.TEXT_HTML, MediaType.APPLICATION_JSON, ExtraMediaType.TEXT_YAML, ExtraMediaType.TEXT_CSV, ExtraMediaType.TEXT_TSV, ExtraMediaType.TEXT_TTL})
     public AttributionView getItemByHex(@PathParam("item-hash") String itemHash) {
-        String sha256Regex = "(sha-256:)(.*)";
+        String sha256Regex = "(" + HashingAlgorithm.SHA256.toString() + ":)(.*)";
```

Comment:
This is an example of a place where you should be using `HashValue.decode(encodedHash)` as the `itemHash` value is an example of an encoded hash that needs to be decoded. The regex logic could even be used in the `HashValue.decode(encodeHash)` method.